requirements.in

```diff
@@ -5,3 +5,4 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 paramiko[gssapi]==3.0.0
+gssapi==1.6.1
```

Comment:
Tested on Slurm and everything was OK with downgraded gssapi version. Paramiko dependencies allow this, so we can pin the version indeed.